robotframework_reportportal/listener.py

```diff
@@ -68,6 +68,7 @@ def log_message(self, message):
         """
         msg = self._build_msg_struct(message)
         logger.debug('ReportPortal - Log Message: {0}'.format(message))
+        msg.handle_img_attachment()
```

Comment:
It should not be here.

Reply:
For real time use case, it needs to be here right? I dont see result_visitor.py getting invoked when robot is launched with --listener argument